adminSDK/directory/index.js

```diff
@@ -116,27 +99,21 @@ function storeToken(token) {
  * @param {google.auth.OAuth2} auth An authorized OAuth2 client.
  */
 function listUsers(auth) {
-  const service = google.admin('directory_v1');
+  const service = google.admin({version: 'directory_v1', auth});
   service.users.list({
-    auth: auth,
     customer: 'my_customer',
     maxResults: 10,
     orderBy: 'email',
-  }, function(err, response) {
-    if (err) {
-      console.log('The API returned an error: ' + err);
-      return;
-    }
-    const users = response.users;
-    if (users.length == 0) {
-      console.log('No users in the domain.');
-    } else {
-      console.log('Users:');
-      for (let i = 0; i < users.length; i++) {
-        const user = users[i];
-        console.log('%s (%s)', user.primaryEmail, user.name.fullName);
-      }
-    }
+  }, (err, {data}) => {
```

Comment:
I think we can't destruct here because data may be nully if there is an `err`.
I think it's best to say `(err, res) => {`